src/cron/bundle.rs

```diff
@@ -278,12 +277,11 @@ fn verify_tx_receipt(tx_receipt: &TxReceipt) -> std::io::Result<bool> {
 
     lazy_static! {
         static ref PUBLIC: PKey<Public> = {
-            let jwk = JWK {
-                kty: "RSA",
-                e: "AQAB",
-                n: std::env::var("BUNDLER_PUBLIC").unwrap(),
-            };
-
+            let jwt_str = format!(
+                "{{\"kty\":\"RSA\",\"e\":\"AQAB\",\"n\":\"{}\"}}",
+                std::env::var("BUNDLER_PUBLIC").unwrap()
+            );
+            let jwk: JsonWebKey = jwt_str.parse().unwrap();
```

Comment:
Same as earlier, why to switch from building the object into creating string representing it and then parsing the it from the string? I don't really see any added value here.

And again, not `jwt_str`, but it should be `jwk_str`.

Reply:
Its being parsed incorrectly. As it was done before, parsing as bytes, it was not generating the jwk correctly.

Reply:
Actually, the mistake is that the value for `n` should be decoded first. This is my mistake in the existing code. Creating JSON string and then parsing it, is an unnecessary step. Instead the code should look like:
```rust
let jwk = JWK {
    kty: "RSA",
    e: "AQAB",
    n: BASE64URL_NOPAD.decode(std::env::var("BUNDLER_PUBLIC").unwrap()),
};
```
Though this code should anyways be changed as we should not read the value in a `lazy_static!`, instead we should be using `KeyManager` where needed.

Reply:
I'm inclined to leave as is for this PR as it works. Create a new PR making this above change. @bluekirby1111 - speak to @mikonieminen if it's unclear as to what you need to do

Reply:
@bluekirby1111 please read all comments carefully. Miko raised this. Change `jwt_str` to `jwk_str`. JWT !== JWK